frontend/ui/widget/opdsbrowser.lua

```diff
@@ -58,7 +58,35 @@ local OPDSBrowser = Menu:extend{
 }
 
 function OPDSBrowser:init()
-    self.item_table = self:genItemTableFromRoot(self.opds_servers)
+    local servers = G_reader_settings:readSetting("opds_servers")
+    if not servers then -- If there are no saved servers, add some defaults
+        servers = {
+          {
+            title = "Project Gutenberg",
+            url =
+            "http://m.gutenberg.org/ebooks.opds/?format=opds",
+          },
+          {
+             title = "Feedbooks",
+             url =
+            "http://www.feedbooks.com/publicdomain/catalog.atom",
+          },
+          {
+             title = "ManyBooks",
+             url = "http://manybooks.net/opds/index.php",
+          },
+          {
+             title = "Internet Archive",
+             url =
+            "http://bookserver.archive.org/catalog/",
+          },
+        }
+        for i in ipairs(servers) do
```

Comment:
This for loop can be replaced with a single line `G_reader_settings:saveSetting("opds_servers", servers)`